spec/spec_helper.rb

```diff
@@ -1,7 +1,11 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
+require 'combustion'
 require 'stitch_fix/log_weasel'
+
+Combustion.initialize! :active_support
```

Comment:
Yeah, sorry I wasn't clear on that. Yes, precisely what you wrote. Combustion ensures that the initializer in the Railtie in log weasel is called, which repro'd the bug that Dean was seeing. 

Noted on not needing to pass any arguments to `Combustion.initialize!` - the docs didn't do a great job of explaining this, and I neglected to dig deeper as I got what I needed out of it.